src/main/java/frc/robot/commands/PlaceCoralAndTakeAlgae.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.ConditionalCommand;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.*;
+
+public class PlaceCoralAndTakeAlgae extends SequentialCommandGroup {
+
+    private static final double speed = -0.2;
+
+    public PlaceCoralAndTakeAlgae(Elevator elevator, AlgaeJoint algaeJoint, Gripper gripper,
+                                  Drivetrain drivetrain, CoralJoint coralJoint, Storage storage,
+                                  Elevator.ElevatorLevel level) {
+        addCommands(new ParallelCommandGroup(
+                        new MoveToHeight(elevator, level),
+                        new RotateStorage(coralJoint, CoralJoint.StoragePose.PLACEMENT),
+                        new RotateAlgaeJointToBottom(algaeJoint)),
+                new ParallelCommandGroup(
+                        new ReleaseCoral(storage), new IntakeAlgae(gripper)),
```

Comment:
the storage should not wait for the algae joint to be at its setpoint in order to release coral and vice versa. call the `andThen()` function